qemu/INSTRUCTIONS.md

```diff
@@ -0,0 +1,53 @@
+# Setting up IlliXR in Qemu
```

Comment:
IlliXR --> ILLIXR throughout this document.

Reply:
Ah yes- my bad!! I thought I fixed this